app/controllers/api/v1/customers_controller.rb

```diff
@@ -106,10 +104,6 @@ def customer_params
         attributes
       end
 
-      def editable_enterprises
-        OpenFoodNetwork::Permissions.new(current_api_user).editable_enterprises.select(:id)
-      end
-
```

Comment:
It looks like a choice has been made for one method over the other. I'm not sure I'm fully understanding this, so I dove in to check:

* `User#enterprises` is a scope for enterprised managed by the user, based on any enterprise **roles** the user has
* `OpenFoodNetwork::Permissions#editable_enterprises` is more complex, based upon more specific permission `:edit_profile` which is assigned by an enterprise **relationship**

Ah ha, I see now :)
So for roles (managed by), I guess it's ok to use the scope directly, but for permissions around relationships, we need the more complicated helper method.